app.js

```diff
@@ -1,8 +1,130 @@
-// import functions and grab DOM elements
+import { renderPoll } from './utils.js';
+
+const inputForm = document.getElementById('input-form');
+
+const decrementA = document.getElementById('decrementA');
+const incrementA = document.getElementById('incrementA');
+const decrementB = document.getElementById('decrementB');
+const incrementB = document.getElementById('incrementB');
+
+const currentPollSection = document.getElementById('current-poll-section');
+// const resultA = document.getElementById('resultA');
+// const resultB = document.getElementById('resultB');
+
+const closePollButton = document.getElementById('close-poll-button');
+
+const closedPollContainer = document.getElementById('closed-poll-container');
```

Comment:
You have very descriptive, clear, naming conventions here.  This goes a long way in making code readable and also helping to avoid confusing yourself when writing code.  Keep it up!